Dockerfile

```diff
@@ -1,35 +1,87 @@
+    Rscript install_pkgs.R r_requirements.txt && \
+    pip3 install --upgrade pip && \
+    pip3 install -r requirements.txt --ignore-installed --user && \
+    pip3 install gunicorn --user
+
+FROM rocker/r-ver:4.0.3 AS prod
+MAINTAINER https://github.com/pacificclimate/chickadee
+LABEL Description="chickadee WPS" Vendor="Birdhouse" Version="0.1.0"
+
 RUN apt-get update && \
     apt-get install -y --no-install-recommends \
-      build-essential \
-      gcc \
-      libgit2-dev \
-      libpq-dev \
       python3.8 \
-      python3-pip \
-      python3-setuptools \
-      python3-dev \
-      libssl-dev \
-      libxml2-dev \
-      libudunits2-dev \
-      libnetcdf-dev && \
-    pip3 install --upgrade pip && \
-    pip3 install -r requirements.txt --ignore-installed && \
-    pip3 install gunicorn && \
-    Rscript install_pkgs.R r_requirements.txt
+      python3-pip
+
+COPY --from=builder /root/.local /root/.local
+
+# Copy compiled library files
+COPY --from=builder /usr/lib/x86_64-linux-gnu/libnetcdf.so.15 \
+  /usr/lib/x86_64-linux-gnu/libhdf5_serial_hl.so.100 \
+  /usr/lib/x86_64-linux-gnu/libhdf5_serial.so.103 \
+  /usr/lib/x86_64-linux-gnu/libcurl-gnutls.so.4 \
+  /usr/lib/x86_64-linux-gnu/libsz.so.2 \
+  /usr/lib/x86_64-linux-gnu/libaec.so.0 \
+  /usr/lib/x86_64-linux-gnu/libudunits2.so.0 \
+  /usr/lib/x86_64-linux-gnu/
+
+# Copy R packages
+# Directories cannot be recursively copied
+COPY --from=builder /root/R/x86_64-pc-linux-gnu-library/4.0/PCICt \
+  /root/R/x86_64-pc-linux-gnu-library/4.0/PCICt
+COPY --from=builder /root/R/x86_64-pc-linux-gnu-library/4.0/udunits2 \
+  /root/R/x86_64-pc-linux-gnu-library/4.0/udunits2
+COPY --from=builder /root/R/x86_64-pc-linux-gnu-library/4.0/ncdf4 \
+  /root/R/x86_64-pc-linux-gnu-library/4.0/ncdf4
+COPY --from=builder /root/R/x86_64-pc-linux-gnu-library/4.0/fields \
+  /root/R/x86_64-pc-linux-gnu-library/4.0/fields
+COPY --from=builder /root/R/x86_64-pc-linux-gnu-library/4.0/foreach \
+  /root/R/x86_64-pc-linux-gnu-library/4.0/foreach
+COPY --from=builder /root/R/x86_64-pc-linux-gnu-library/4.0/seas \
+  /root/R/x86_64-pc-linux-gnu-library/4.0/seas
+COPY --from=builder /root/R/x86_64-pc-linux-gnu-library/4.0/abind \
+  /root/R/x86_64-pc-linux-gnu-library/4.0/abind
+COPY --from=builder /root/R/x86_64-pc-linux-gnu-library/4.0/ClimDown \
+  /root/R/x86_64-pc-linux-gnu-library/4.0/ClimDown
+COPY --from=builder /root/R/x86_64-pc-linux-gnu-library/4.0/doParallel \
+  /root/R/x86_64-pc-linux-gnu-library/4.0/doParallel
+COPY --from=builder /root/R/x86_64-pc-linux-gnu-library/4.0/maps \
+  /root/R/x86_64-pc-linux-gnu-library/4.0/maps
+COPY --from=builder /root/R/x86_64-pc-linux-gnu-library/4.0/spam \
+  /root/R/x86_64-pc-linux-gnu-library/4.0/spam
+COPY --from=builder /root/R/x86_64-pc-linux-gnu-library/4.0/dotCall64 \
+  /root/R/x86_64-pc-linux-gnu-library/4.0/dotCall64
+COPY --from=builder /root/R/x86_64-pc-linux-gnu-library/4.0/iterators \
+  /root/R/x86_64-pc-linux-gnu-library/4.0/iterators
```

Comment:
Similar comment to above, it would be helpful to know how this list was put together. Is there some way to have this command in a single layer? To my eye it seems unlikely.

Reply:
I tried copying recursively, but it won't work for the directories. Also, I manually ran every process to figure out what packages are used.